source/fundamentals/crud/read-operations/project.txt

```diff
@@ -57,149 +74,132 @@ method in the options of the following read operation methods:
 Exclude a Field
 ~~~~~~~~~~~~~~~
 
-To exclude a field, pass the field you want to exclude and a ``0`` to
-the ``SetProjection()`` method. For all fields you don't explicitly
-list in the projection, the driver includes them.
+To exclude a field, pass the field you want to exclude with a ``0`` to
+the ``SetProjection()`` method. The driver includes all fields that are
+not explicitly excluded in the projection document.
 
 Example
 ```````
 
-The following example excludes the ``rating`` from the matched documents
-from the ``Find()`` method:
+The following example excludes the ``course_id`` and ``enrollment``
+fields from the matched documents returned by the ``Find()`` method:
 
 .. io-code-block::
    :copyable: true
 
    .. input::
       :language: go
 
-      opts := options.Find().SetProjection(bson.D{{"rating", 0}})
-
-      cursor, err := coll.Find(context.TODO(), bson.D{}, opts)
+      filter := bson.D{}
+      opts := options.Find().SetProjection(bson.D{{"course_id", 0}, {"enrollment", 0}})
+      
+      cursor, err := coll.Find(context.TODO(), filter, opts)
       if err != nil {
-         panic(err)
+          panic(err)
       }
-
-      var results []bson.D
+      
+      var results []Course
       if err = cursor.All(context.TODO(), &results); err != nil {
-         panic(err)
+          panic(err)
       }
+
       for _, result := range results {
-         fmt.Println(result)
+          res, _ := bson.MarshalExtJSON(result, false, false)
+          fmt.Println(string(res))
       }
 
    .. output::
       :language: none
       :visible: false
 
-      //results truncated
-      [{_id ObjectID("...")} {type Masala}]
-      [{_id ObjectID("...")} {type Assam}]
-      [{_id ObjectID("...")} {type Oolong}]
-      [{_id ObjectID("...")} {type Earl Grey}]
-      [{_id ObjectID("...")} {type English Breakfast}]
+      {"title":"Primate Behavior"}
+      {"title":"Revolution and Reform"}
 
 Include a Field
 ~~~~~~~~~~~~~~~
 
-To include a field, pass the field you want to include and a ``1`` to
-the ``SetProjection()`` method. For all fields you don't explicitly
-list in the projection, the driver excludes them.
-
-.. important::
-
-    You can exclude the ``_id`` field even if you specified to include
-    certain fields. By default, the driver includes the ``_id`` field.
-    You must explicitly exclude the ``_id`` field if you do not want it
-    returned.
+To include a field, pass the field you want to include with a ``1`` to
+the ``SetProjection()`` method. The driver excludes all fields that are
+not explicitly included in the projection document.
```

Comment:
Question:

Now I am very confused. I feel like something is missing from this description and above. Because we now both say that we include all fields not explicitly excluded and that we exclude all fields not explicitly included. These cannot both be true. 

Does this need something additional like "...if you specify any fields to include"?